src/MCPServer/lib/assets/workflow.json

```diff
@@ -2705,6 +2816,25 @@
         "sv": "Normalisera"
       }
     },
+    "1dad74a2-95df-4825-bbba-dca8b91d2371": {
+      "config": {
+        "@manager": "linkTaskManagerChoice",
+        "@model": "MicroServiceChainChoice",
+        "chain_choices": [
+          "1ac7d792-b63f-46e0-9945-d48d9e5c02c9",
+          "697c0883-798d-4af7-b8b6-101c7f709cd5"
+        ]
+      },
+      "description": {
+        "en": "Do you want to scan for viruses in extracted files?"
+      },
+      "exit_codes": {},
+      "fallback_job_status": "Failed",
+      "fallback_link_id": null,
+      "group": {
+        "en": "Extract packages"
```

Comment:
Should we include the rest of existing translations (e.g. `es`, `fr`, etc) for the `Extract packages` group here?